tests/common.go

```diff
@@ -7144,6 +7144,34 @@ func CreateAzureBucket(bucketName string, objectlock bool, mode Mode, retentionD
 
 }
 
+// UpdateConfigmap creates or updates an existing configmap with the given data
+func UpdateConfigmap(configMapName string, data map[string]string) error {
+	configmap, err := k8sCore.GetConfigMap(configMapName, "default")
```

Comment:
nit: You can define a const for namespace and set it to default since it is used twice

Reply:
I'll fix it for this function but I see it's repeated multiple times we'll need to clean that up someday.